docs/sdk-and-tools/mxpy/mxpy-cli.md

```diff
@@ -0,0 +1,16 @@
+---
+id: mxpy-cli
+title: mxpy CLI
+---
+
+mxpy's Command Line Interface
+
+**mxpy** exposes a number **commands**, organized within **groups**. The complete Command Line Interface is listed [here](https://github.com/multiversx/mx-sdk-py-cli/blob/main/mxpy/CLI.md). Command usage and description are available through the `--help` flags.
```

Comment:
The link to the CLI commands will not work because the package is not named `mxpy`. The package is named `multiversx_sdk_cli`, so it should be: `https://github.com/multiversx/mx-sdk-py-cli/blob/main/multiversx_sdk_cli/CLI.md

Reply:
I've moved `CLI.md` (and `CLI.md.sh` one level up in the repository), and adjusted the link.